### DISCUSSION TITLE
[Projects Beta] cannot add new milestone

### DISCUSSION THREAD

**Discussion by zufardhiyaulhaq** [community/community#9280] [Projects and Issues]
![gambar](https://user-images.githubusercontent.com/11990726/147348438-2d3bebf9-2bc1-4a0d-9e33-1f63cb571d3c.png)

cannot create milestone

**Comment by MarkBennett** [comment]
I agree. Milestones are a great way to communicate progress to the team, so having a quick way to create them from the boards would be handy.
